tests/b2b-mp/storefront/tests/checkout/B2B-MP-S3-checkout-1-item.js

```diff
@@ -35,3 +39,7 @@ options.thresholds[`http_req_duration{url:${storefrontBaseUrl}/en/checkout/succe
 export function executeCheckoutScenario() {
     checkoutScenario.execute();
 }
+
+export function handleSummary(data) {
+    return SummaryHelper.handleSummary(data, environment, testId);
```

Comment:
can we use stmth like this? in order to avoid code duplication? https://community.grafana.com/t/global-handlesummary/119406/5